app/errors.py

```diff
@@ -0,0 +1,14 @@
+class VaccineError(Exception):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
```

Comment:
Consider adding a descriptive error message to your `NotVaccinatedError` class. For example, `class NotVaccinatedError(VaccineError):
    def __init__(self, message="The person is not vaccinated."):
        super().__init__(message)`